README.md

```diff
@@ -1 +1,62 @@
-# Test-review-to-do-list-ob
\ No newline at end of file
+# to-do-list-OB
+In this project, I will build a simple HTML list of To Do tasks.  This simple web page will be built using webpack and served by a webpack dev server.
+
+## learning Objectives:
+
+- Use webpack to bundle JavaScript.
+- Learn how to use proper ES6 syntax.
+- Use ES6 modules to write modular JavaScript.
+
+## Live Demo
+
+[Live Demo Link](https://osvaldobc.github.io/trafficConferencePage/)
+
+
+## interactions
+
+
+## Getting Started
+
+The steps to clone and the project locally is :
+git clone git@github.com:OsvaldoBC/to-do-list-OB.git
+
+
+## Built With
+
+- HTML
+- CSS
+- JS
+
+### Prerequisites
+
+No prerequisite required
+
+
+## Authors
+
+👤 ** Osvaldo Jose Barrios Cortina **
+
+- Twitter : [@OsvaldoBarrio20](https://twitter.com/OsvaldoBarrio20)
+- GitHub  : [@OsvaldoBC](https://github.com/OsvaldoBC)
+- LinkedIn: [osvaldo-barrios-data-science](https://linkedin.com/in/osvaldo-barrios-data-science)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Inspiration.....
+- etc....
+
+
```

Comment:
if you have any Acknowledgments for this project you cant put it otherwise you can erase it